pkg/openevec/eden.go

```diff
@@ -366,29 +371,24 @@ func setupEdenScripts(cfg EdenSetupArgs) error {
 }
 
 func setupConfigDir(cfg EdenSetupArgs, eveConfigDir, softSerial, zedControlURL string, grubOptions []string) error {
-	if _, err := os.Stat(filepath.Join(cfg.Eden.CertsDir, "root-certificate.pem")); os.IsNotExist(err) {
```

Comment:
why regenerate certificates on every setup? I'm not against that, just wondering why

Reply:
Reverted that :)